### PR TITLE
feat: add new chains [APE-941]

### PR DESCRIPTION
**Pull request by bilbeyt**
### What I did

Added base and polygon zkevm

fixes: #

### Checklist

- [ ] Passes all linting checks (pre-commit and CI jobs)
- [ ] New test cases have been added and are passing
- [ ] Documentation has been updated
- [ ] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

**Review by antazoey — DISMISSED** [review]
thanks!

if this works for you, then we are good, consider adding tests and docs. See past PRs that add networks like this.

**Comment by bilbeyt** [comment]
> thanks!
> 
> if this works for you, then we are good, consider adding tests and docs. See past PRs that add networks like this.

I didnt try it as i am waiting to merge my new plugins first. I guess without them i cant use these properly.

**Comment by antazoey** [comment]
> I didnt try it as i am waiting to merge my new plugins first. I guess without them i cant use these properly.

OK let us know how it goes.

File: ape_etherscan/utils.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
     "ethereum": "ETHERSCAN_API_KEY",
     "fantom": "FTMSCAN_API_KEY",
     "optimism": "OPTIMISTIC_ETHERSCAN_API_KEY",
+    "polygon-zkevm": "POLYGON_ZKEVM_API_KEY",
```
Comment (antazoey) [inline ape_etherscan/utils.py:6]:
```suggestion
    "polygon-zkevm": "POLYGONSCAN_ZKEVM_API_KEY",
```

Comment (antazoey) [reply ape_etherscan/utils.py:6]: or something with `scan` in it?

File: ape_etherscan/utils.py [hunk 2]
```diff
@@ -3,6 +3,8 @@
...
+    "base": "BASE_API_KEY",
```
Comment (antazoey) [inline ape_etherscan/utils.py:7]:
```suggestion
    "base": "BASESCAN_API_KEY",
```

just basing these off the web service names, like all the others

File: ape_etherscan/utils.py [hunk 3]
```diff
@@ -25,6 +27,13 @@
         "mainnet",
         "goerli",
     ],
+    "base": [
+        "goerli",
```
Comment (antazoey) [inline ape_etherscan/utils.py:31]: Missing mainnet? I see it referenced above?

File: ape_etherscan/utils.py (hunk 1 above)
Comment (bilbeyt) [reply ape_etherscan/utils.py:6]: will go with POLYGON_ZKEVM_ETHERSCAN_API_KEY

**Comment by bilbeyt** [comment]
> > I didnt try it as i am waiting to merge my new plugins first. I guess without them i cant use these properly.
> 
> OK let us know how it goes.

Tried verifications on both polygon zkevm and base goerli using packages locally and they are working fine.

**Comment by bilbeyt** [comment]
@antazoey can we merge this one as we already transferred ape-base and ape-polygon-zkevm to apeworx organisation? this PR as long as i tried works

**Comment by antazoey** [comment]
> @antazoey can we merge this one as we already transferred ape-base and ape-polygon-zkevm to apeworx organisation? this PR as long as i tried works

Yes, I can do all this right now

**Comment by antazoey** [comment]
moving to https://github.com/ApeWorX/ape-etherscan/pull/91 because your org prevents us from collaborating.

**Pull request closed without merging**